apis/config/v1alpha1/defaults.go

```diff
@@ -72,6 +74,18 @@ func SetDefaults_LandscaperConfiguration(obj *LandscaperConfiguration) {
 	if obj.DeployerManagement.Agent.OCI == nil {
 		obj.DeployerManagement.Agent.OCI = obj.Registry.OCI
 	}
+
+	if obj.DeployerManagement.DeployerRepositoryContext == nil {
+		if obj.Controllers.Contexts.Config.Default.RepositoryContext != nil {
+			obj.DeployerManagement.DeployerRepositoryContext = obj.Controllers.Contexts.Config.Default.RepositoryContext.DeepCopy()
+		} else if obj.RepositoryContext != nil {
+			obj.DeployerManagement.DeployerRepositoryContext = obj.RepositoryContext.DeepCopy()
+		} else {
+			defaultCtx, _ := cdv2.NewUnstructured(cdv2.NewOCIRegistryRepository("eu.gcr.io/gardener-project/development", ""))
```

Comment:
Wouldn't it be safer to have the previous default repository context (eu.gcr.io/gardener-project/development) when none is set explicitely?
Will this be backwards compatible?

Reply:
I have changed this accordingly